website/docs/reference/macro-properties.md

```diff
@@ -4,7 +4,7 @@ title: Macro properties
 
 Macro properties can be declared in `.yml` files in:
 - your `macros/` directory (as defined by the [`macro-paths` config](macro-paths))
-- your `models/` directory (as defined by the [`source-paths` config](source-paths))
+- your `models/` directory (as defined by the [`model-paths` config](model-paths))
```

Comment:
as above/elsewhere, this isn't wrong exactly but it does play into a common misconception